pkg/kube/inject/testdata/webhook/TestWebhookInject_podRedirectAnnot_template.yaml

```diff
@@ -0,0 +1,26 @@
+policy: enabled
+alwaysInjectSelector: []
+neverInjectSelector: []
+injectedAnnotations: {}
```

Comment:
Why do we need `podRedirectAnnot` if we already have `injectedAnnotations`?

Reply:
I vaguely remember noticing that {{podRedirectAnnot}} is templatable, whereas {{injectedAnnotations}} isn't, so I assumed that was why `podRedirectAnnot` was introduced. Since kube-inject used {{podRedirectAnnot}} I just added the same code that injects it into webhook in my original PR.

Reply:
Good point. So `injectedAnnotations` is templated, but its done at install time whereas `podRedirectAnnot` is done at injection time. However, its not clear why we actually need information at injection time, given it looks like its allow just pulling it from install time settings (and anything it *did* need from the pod, CNI has access to the full pod spec).